db/migrate_cards/data/production_export_slim.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "name": "production export",
+    "type": "Pointer"
+  }
```

Comment:
meta information like name, type or codename is still json. Could eventually move to the migration but that probably means that the migration generator has to do the export instead of the rake task (or the more hacky way: the rake import task has to edit the migration file)